src/Emulator/Visuals/Visuals.js

```diff
@@ -1,11 +1,236 @@
-import React from 'react';
+import React, {useRef, useState, useEffect} from 'react';
 import './Visuals.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearchPlus, faSearchMinus, faExpand, faEyeSlash} from '@fortawesome/free-solid-svg-icons';
+import Draggable from 'react-draggable';
+import { parse } from 'path';
 
 const Visuals = () => {
+  const containerHeight = '50vh';
+  const containerWidth = '75vw';
+  const additionalImageSpace = 20;
+
+  const [zoom, setZoom] = useState(true);
+  const [hideImage, setHideImage] = useState(false);
+  const [toggleMax, setMax] = useState(false);
+  const imageRef = useRef();
+  const [containerDimensions, setContainerDimensions] = useState({height: containerHeight, width: containerWidth});
+  const [contentDimensions, setContentDimensions] = useState({height: 0, width: 0});
+  const [image, setImage] = useState('null');
+  const [, setSrc] = useState('');
+  const [, setDrags] = useState(0);
+  const defaultX = 200;
+  const defaultY = 200;
```

Comment:
Why `200`? Can they be removed? If not, please define them at the top of the file as constants to avoid "magic numbers"

Reply:
it's just the general starting point of the element when it's initially rendered onto the image panel